packages/kubernetes-1.15/kubernetes-1.15.spec

```diff
@@ -87,6 +87,9 @@ install -m 0644 %{S:7} %{buildroot}%{_cross_templatedir}/kubelet-bootstrap-kubec
 install -d %{buildroot}%{_cross_tmpfilesdir}
 install -p -m 0644 %{S:8} %{buildroot}%{_cross_tmpfilesdir}/kubernetes.conf
 
+install -d %{buildroot}%{_cross_libexecdir}/kubernetes
+ln -rs %{_sharedstatedir}/kubelet/plugins %{buildroot}%{_cross_libexecdir}/kubernetes/kubelet-plugins
```

Comment:
You mentioned offline that this results in 10 levels of `../` in the link target, which may be due to expanding `%{buildroot}` on one side and not the other, or something like that.  (There are 5 levels in the expansion of `%{_cross_libexecdir}`, so maybe that's telling?)  I'm curious if it could be simplified somehow.  It might help to show the links in the testing description so it's easier to understand the structures being linked together.